metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -1060,6 +1079,18 @@ def get_connection_object_page(
                         context=f"{permission_mode_errors}",
                     )
 
+                if node_limit_errors:
+                    logger.debug(f"Node Limit Error. query_data {query_data}")
+                    self.report.warning(
+                        title="Node Limit Errors",
```

Comment:
please improve the error messaging. an end user won't know what "node limit errors" means

what is impacted by this kind of error?

Reply:
done